packages/xo-server/sample.config.toml

```diff
@@ -157,3 +157,8 @@ port = 80
 #
 # Default: false
 #useSudo = false
+
+# Debounce the fetch of the metadata backups
+#
+# Default: 6e4
+# fetchBackupsDelay = 6e4
```

Comment:
- add the default configuration in `config.toml`
- remove from `sample.config.toml` (advanced setting, no need to expose it to users, those who need it will be able to find it)
- use `parseDuration` to allow setting a human value (e.g. `'1 min'`)
- create a `backup` section
- rename the setting to `listingDebounce`